src/main/java/github/grace000/erversay/Constants.java

```diff
@@ -0,0 +1,15 @@
+package github.grace000.erversay;
+
+public final class Constants {
+    public final static String DEFAULT_VERSION = "HTTP/1.1";
+    public final static String DEFAULT_CODE = "200";
+    public final static String DEFAULT_STATUS = "OK";
+    public final static String CODE = "code";
+    public final static String STATUS = "status";
+    public final static String NOT_FOUND_CODE = "404";
+    public final static String NOT_FOUND_STATUS = "Not Found";
+    public final static String SIMPLE_GET_URI = "/simple_get";
+    public final static String SP = " ";
+    public final static String CRLF = "\r\n";
+    public final static String DOUBLE_LINE_FEED = "\r\n\r\n";
```

Comment:
Why not `CRLF + CRLF`?

Reply:
😅I changed that...I think I got carried away there.